src/screens/Customers/CustomerMain.tsx

```diff
@@ -26,4 +21,9 @@ function CustomerMain(props: RouteComponentProps) {
   );
 }
 
-export default CustomerMain;
+const mapStateToProps = (state: RootState) => {
+  let customers = state.siteData.currentSite.customers;
```

Comment:
just to ensure we're defensive w/code, let's do null checks to make sure we have a `currentSite` and whatnot (and if not customers is an empty list so the rest of the code doesnt break)

Reply:
I saw that this file changed a lot in main, should I still make this change?